src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/SafeEvpPKeyHandle.OpenSsl.cs

```diff
@@ -177,5 +193,84 @@ public static SafeEvpPKeyHandle OpenPublicKeyFromEngine(string engineName, strin
 
             return Interop.Crypto.LoadPublicKeyFromEngine(engineName, keyId);
         }
+
+        /// <summary>
+        ///   Open a named public key using a named OpenSSL <code>provider</code>.
+        /// </summary>
+        /// <param name="providerName">
+        ///   The name of the <code>provider</code> to process the key open request.
+        /// </param>
+        /// <param name="keyUri">
+        ///   The URI of the key to open.
```

Comment:
Should the paramdoc try to indicate that the URI is determined by the provider, not just some random file to load off teh interwebz?

```suggestion
        ///   The provider-assigned URI of the key to open.
```

(or something?)

Sure, the demo provider can load any pkcs8 key on the mounted filesystem via the file scheme... but that URI scheme is still something it made up.